nixos/modules/config/pulseaudio.nix

```diff
@@ -312,6 +312,7 @@ in {
         serviceConfig = {
           Type = "notify";
           ExecStart = "${binaryNoDaemon} --log-level=${cfg.daemon.logLevel} --system -n --file=${myConfigFile}";
+          ExecStop = "${binaryNoDaemon} --kill";
```

Comment:
are you using systemWide pulseaudio? Because that isn't the default in nixos and is greatly discouraged in use
(this is in the mkIf systemWide block)

Reply:
no, I am not:
```
  hardware.pulseaudio = {
    support32Bit = true;
    enable = true;
  };
```

Reply:
I'll have to do some more investigating.. maybe these changes just coincided with something else